src/languageProvider/messages/en-US.js

```diff
@@ -0,0 +1,54 @@
+export default {
+  common: {
```

Comment:
Please organize the messages per feature. For example;

login: {
    // All messages from login form
},
signup: {
    // All messages from signup form
}